apis/Google.Devtools.Aspnet/Google.Devtools.Aspnet/CloudErrorReportingExceptionLogger.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Clouderrorreporting.v1beta1;
```

Comment:
You appear to be using the REST API - I'd have expected to use the gRPC one. Any reason for this?